src/internal.rs

```diff
@@ -65,3 +66,24 @@ pub union Dav1dTaskContext_cf {
     pub cf_8bpc: [int16_t; 1024],
     pub cf_16bpc: [int32_t; 1024],
 }
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub struct Dav1dTaskContext_scratch_interintra_edge_8 {
+    pub interintra_8bpc: [uint8_t; 4096],
+    pub edge_8bpc: [uint8_t; 257],
+}
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub struct Dav1dTaskContext_scratch_interintra_edge_16 {
+    pub interintra_16bpc: [uint16_t; 4096],
+    pub edge_16bpc: [uint16_t; 257],
+}
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+pub union Dav1dTaskContext_scratch_interintra_edge {
```

Comment:
Minor requestion: Could you instead name this struct `Dav1dTaskContext_scratch`? I think the name currently is a bit unnecessarily verbose. Likewise the other two could be shortened to `Dav1dTaskContext_scratch_8bpc` and `Dav1dTaskContext_scratch_16bpc`.

Reply:
Actually nevermind, I see now that this struct more deeply nested than I thought (originally I thought this struct was the `scratch` field in `Dav1dTaskContext`, but looks like it's actually one of the anonymous inner structs).

Reply:
Yeah, I was planning on naming the outer one `Dav1dTaskContext_scratch`, as this one's more deeply nested.